test/src/test/java/hudson/cli/BuildCommandTest.java

```diff
@@ -195,7 +203,7 @@ public void refuseToBuildDisabledProject() throws Exception {
         CLICommandInvoker invoker = new CLICommandInvoker(j, new BuildCommand());
         CLICommandInvoker.Result result = invoker.invokeWithArgs("the-project");
 
-        assertThat(result, failedWith(4));
+        assertThat(result, failedWith(CLIReturnCodeStandard.ILLEGAL_STATE.getCode()));
```

Comment:
If you add another matcher with the same name `failedWith`, you could match against the `CLIResultCode` itself which might be more expressive.

Reply:
It's the goal to add it in the JTH but I do not want to have a too strong link between the core depencency and the JTH to avoid troubles (like the requirement to release both at the same time).